packages/components/src/form-toggle/styles/form-toggle-styles.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+/**
+ * Internal dependencies
+ */
+import { color, reduceMotion, rtl } from '../../utils';
+import {
+	toggleBorderWidth,
+	toggleWidth,
+	toggleHeight,
+	appearTransform,
+} from './utils';
+
+export const StyledToggleTrack = styled.span`
+	content: '';
+	display: inline-block;
+	box-sizing: border-box;
+	vertical-align: top;
+	background-color: ${ color( 'white' ) };
+	border: ${ toggleBorderWidth }px solid ${ color( 'darkGray.primary' ) };
+	width: ${ toggleWidth }px;
+	height: ${ toggleHeight }px;
+	border-radius: ${ toggleHeight / 2 }px;
+	transition: 0.2s background ease;
+	${ reduceMotion( 'transition' ) }
+`;
+
+export const StyledToggleThumb = styled.span`
+	display: block;
+	position: absolute;
+	box-sizing: border-box;
+	top: ${ toggleBorderWidth * 3 }px;
+	${ rtl( { left: `${ toggleBorderWidth * 3 }px` } ) };
+	width: ${ toggleHeight - toggleBorderWidth * 6 }px;
+	height: ${ toggleHeight - toggleBorderWidth * 6 }px;
+	border-radius: 50%;
+	transition: 0.1s transform ease;
+    ${ reduceMotion( 'transition' ) }
+	background-color: ${ color( 'darkGray.primary' ) };
+	border: 5px solid ${ color(
+		'darkGray.primary'
+	) }; // Has explicit border to act as a fill in Windows High Contrast Mode.
+`;
+
+export const StyledToggleInput = styled.input`
+	position: absolute;
+	top: 0;
+	${ rtl( { left: 0 } ) };
+	width: 100%;
+	height: 100%;
+	opacity: 0;
+	margin: 0;
+	padding: 0;
+	z-index: z-index( '.components-form-toggle__input' );
```

Comment:
We'll need to change this :). This is probably a Sass mixin.
To start, we can create a variable with the `zIndex` (similar to what you did with others in `utils`).

There's going to be some disconnect between the Sass values and JS values during this migration process.

Hopefully we can introduce a JS based zIndex system - perhaps something similar to the one within G2 Components:
https://github.com/ItsJonQ/g2/blob/master/packages/styles/src/mixins/zIndex.js